src/main/java/com/lazerycode/jmeter/configuration/JMeterArgumentsArray.java

```diff
@@ -207,8 +207,10 @@ public void setTestFile(File value) {
 					argumentsArray.add(overrideRootLogLevel.toString());
 					break;
 				case PROPFILE2_OPT:
-					argumentsArray.add(PROPFILE2_OPT.getCommandLineArgument());
-					argumentsArray.add(customPropertiesFile);
+					for (String customPropertiesFile : customPropertiesFiles) {
+						argumentsArray.add(PROPFILE2_OPT.getCommandLineArgument());
+						argumentsArray.add(customPropertiesFile);
+					}
```

Comment:
this will produce a command line that look like:

`-q custom1.properties -q custom2.properties`

Should it be this, or:

`-q custom1.properties custom2.properties`